gateway/ln-gateway/src/lib.rs

```diff
@@ -368,13 +373,22 @@ impl Gateway {
         let task_group = TaskGroup::new();
         task_group.install_kill_handler();
 
+        let gateway_id = Self::load_or_create_gateway_id(&gateway_db).await;
+
+        // for JWT it is necessary to create an encoding secret that will be used each
+        // time a new JWT token is generated. The encoding secret ensures token's
+        // integrity and authenticity, so it is necessary to use a secure random
+        // number generator to generate strong keys.
+        let encoding_secret: [u8; 16] = rand::thread_rng().gen();
```

Comment:
The secret being genearted on the fly means a GW restart invalidates old tokens  I assume? Not a problem imo, just want to point it out so everyone is aware.

Reply:
Yes, you are right. Where should I put a note about that? Would it be okay as an additional comment, or should I create an additional document for that?